src/sentry_plugins/amazon_sqs/plugin.py

```diff
@@ -4,7 +4,6 @@
 
 from sentry.plugins.bases.data_forwarding import DataForwardingPlugin
 from sentry.utils import json
-
```

Comment:
Unfortunately there's no way to specify separate 1st party module groups. I think this is fair to group sentry and sentry_plugins though.